modules/javafx.base/src/test/java/test/javafx/util/DurationTest.java

```diff
@@ -286,47 +286,47 @@
 
     @Test public void add_ZERO_and_INDEFINITE_ResultsInIndefinite() {
         //assertTrue(0.0 + Double.POSITIVE_INFINITY == Double.POSITIVE_INFINITY); // sanity check
-        assertEquals(new Double(Double.POSITIVE_INFINITY), new Double(0.0 + Double.POSITIVE_INFINITY)); // sanity check
+        assertEquals(Double.valueOf(Double.POSITIVE_INFINITY), Double.valueOf(0.0 + Double.POSITIVE_INFINITY)); // sanity check
```

Comment:
I don't understand why convert to `Double` for the assertion test, but more than that, I don't understand why this test is needed for `Duration`. Isn't this is a guaranteed behavior of fp arithmetic?

Same for all other such asserts.

Reply:
From the look of these tests, many look unnecessary to be here. But As these tests exist from very beginning, there might have been a good reason that they were added. As long as they don't harm, and if you don't have any hard call on them, let's keep them as is.(with just the `valueOf` change)